solutions/CSES Planets Cycles.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: cses-1751
+source: CSES 
+title: Planets Cycles
+author: Maggie Liu
+---
+
+Define the number of teleportations starting from a planet as the $\texttt{cyclelength}$ of that planet. For each planet that hasn't been visited, we want to find its $\texttt{cyclelength}$. Call the planet we are performing $\texttt{dfs}$ from the $\textit{start}$. As we perform $\texttt{dfs}$ from the $\textit{start}$, keep track of the planets seen, in order, in the $\texttt{cycle}$ queue. When we reach a planet that has already been visited (call this planet the $\textit{repeat}$), we add the $\texttt{cyclelength}$ of the $\textit{repeat}$ to the current $\texttt{step}$ count. 
```

Comment:
No mention of variable "step" when visiting new nodes. Maybe add smth along the lines of "... keep track of $\texttt{step}$, the length of the path so far... "

then add "... count because we would continue to visit all of the nodes that $\textit{repeat}$ would go on to visit." to the end

Reply:
Oh you define step below. I still think it would help to mention it in paragraph 1 tho